src/adapter/graphql/src/scalars/odf_generated.rs

```diff
@@ -1418,6 +1418,51 @@ impl From<odf::metadata::TransformSql> for TransformSql {
 // https://github.com/kamu-data/open-data-fabric/blob/master/open-data-fabric.md#transforminput-schema
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#[derive(Interface, Debug, Clone)]
```

Comment:
Comment for readers: yes, it's a change to the generated code. 

A little later, the generator code will be updated.